tailwind.config.js

```diff
@@ -1,5 +1,4 @@
 // autogenerated and mostly unchanged,
```

Comment:
```suggestion
// autogenerated and mostly unchanged,
// except for custom color theme
```
das sollte da eigentlich schon noch bleiben :sweat_smile: nur der darkMode-Teil weg

Reply:
als ob ich die Zeile darüber auch mit gelöscht habe 😂 
Heute ist wohl nen small 🧠 Tag tf 😂

Reply:
das war auf einer Zeile, ich wollte, dass du nur nen Teil einer Zeile löschst